docs/helm-chart/index.rst

```diff
@@ -140,3 +140,22 @@ will not start as the migrations will not be run:
 This is so these CI/CD services can perform updates without issues and preserve the immutability of Kubernetes Job manifests.
 
 This also applies if you install the chart using ``--wait`` in your ``helm install`` command.
+
+.. note::
+    While deploying this Helm chart with Argo, you might encounter issues with ``DB Migrations`` not running automatically on upgrade.
+
+To ensure DB Migrations with Argo CD, you will need to:
+
+.. code-block::yaml
+
+    migrateDatabaseJob:
+        jobAnnotations:
+            "argocd.argoproj.io/hook": Sync
+
+and, equivalent configurations in case of Flux CD
```

Comment:
Is the code different for Flux CD? If so, could you provide an example?

Reply:
@BasPH tbh, I haven't used Flux CD, nor could I find anything on how to achieve this in Flux CD docs. Do you think its more appropriate to remove this line? I only put it in there because the original section mentioned Flux CD in the heading.

Reply:
My intent was to document:

- That there are 'specific' extra steps to make this helm chart work with Argo and Flux
- Specifically, resolve the documentation wrt to Argo because we have been getting a few issues mentioning helm-chart doesn't work with Argo

Reply:
I'd suggest we drop this line if we don't know what we need for flux.